src/bp-activity/bp-activity-functions.php

```diff
@@ -993,6 +993,18 @@ function bp_activity_add_user_favorite( $activity_id, $user_id = 0 ) {
 		 */
 		do_action( 'bp_activity_add_user_favorite', $activity_id, $user_id );
 
+		// Add user reaction.
+		if ( ! empty( $activity_id ) && function_exists( 'bb_load_reaction' ) ) {
+			$reaction_id = (int) bp_get_option( 'bb_reactions_default_like_reaction_added' );
```

Comment:
@jitendrabanjara1991 Create one global function to retrieve the reaction id instead of using everywhere `$reaction_id = (int) bp_get_option( 'bb_reactions_default_like_reaction_added' );` so in future if we make a changes you don't need to go everywhere to make the changes.